src/components/Beers/Beers.styled.ts

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+import { white } from '../../sharedStyle/styles' 
+import { 
+  StyledH1, 
+  StyledI, 
+  StyledInput, 
+  StyledLabel, 
+  StyledForm, 
+  StyledInputGroup, 
+  StyledButtonGroup, 
+} from '../../sharedStyle/styles';
+
+export const StyledBeersContent = styled.div`
+display: flex;
+flex-direction: column;
+color: ${white};
+`
+export const StyledTitle = styled(StyledH1)``;
```

Comment:
Maybe it's a good idea to export a `StyledTitle` that extends `StyledH1` but I'm not sure about doing by default `styled(StyledH1)`` `, maybe just `export const StyledTitle = StyledH1;` is enough as long as we don't want to customize it. I'm not sure about this though, What do you think ?

Reply:
I agree, i just didnt had the idea ;) Code is updated (see last commit)